lcls-twincat-common-components/Library/Devices/PPM/FB_PPM_R_PowerMeter.TcPOU

```diff
@@ -0,0 +1,306 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1">
+  <POU Name="FB_PPM_R_PowerMeter" Id="{fedad4c7-7971-4466-a86d-d67d48d0e596}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_PPM_R_PowerMeter
+VAR_INPUT
+    fTempSP: REAL;
+    bVetoTempFFO: BOOL;
+    sDeviceName: STRING;
+
+    // Used to know when to stop background auto-collection
+    eEnumGet: E_PPM_States;
+    stYAxisState: DWORD;
+
+    // Calibrated for each gentec, should be passed as input or set via pv
```

Comment:
I think values set via PV will always be overridden cyclically by the VAR_INPUT variable in prod, but perhaps this should be a PLC-side parameter anyway?

Reply:
I made it input only so it can be displayed on the lucid screen